### PR TITLE
Change text

### PR DESCRIPTION
**Pull request by ambuznego**
image
image

### CODE REVIEW

**Review by vicmgs — APPROVED** [review]

File: assets/js/components/common/DashboardLayout.jsx [hunk 1]
```diff
@@ -171,7 +179,7 @@ class DashboardLayout extends Component {
                       </a>
                     ))}
                     <div style={{ marginLeft: "auto" }}>
-                      © 2021 Helium Systems Inc.
+                      © 2022 Helium Systems Inc.
```
Comment (JayKickliter) [inline assets/js/components/common/DashboardLayout.jsx:182]: If incrementing the date, then should it be Nova?

Comment (ambuznego) [reply assets/js/components/common/DashboardLayout.jsx:182]: Will defer to @jdgemm

Comment (JayKickliter) [reply assets/js/components/common/DashboardLayout.jsx:182]: Yeah, I don't really know, but figured Helium Systems Inc. wasn't it

Comment (ambuznego) [reply assets/js/components/common/DashboardLayout.jsx:182]: per Dal, keeping as is

**Pull request merged**